Libraries/acdriver/Sources/CompileAction.cpp

```diff
@@ -239,12 +239,27 @@ WarnUnsupportedOptions(Options const &options, Result *result)
     }
 }
 
+static bool
+CheckInvalidOptions(Options const &options, Result *result)
```

Comment:
Nit:  Name this to something that's more obviously boolsy like "areOptionsValid" -- otherwise it's not very clear to me what it means if it returns true (is the functiuon checking for the presence of invalid options but it didn't find any, or is the check supposed to fail *if* invalid options are present?).